server/src/main/java/org/opensearch/action/admin/indices/stats/IndicesStatsResponse.java

```diff
@@ -69,7 +69,7 @@ public class IndicesStatsResponse extends BroadcastResponse {
         shards = in.readArray(ShardStats::new, (size) -> new ShardStats[size]);
     }
 
-    IndicesStatsResponse(
+    public IndicesStatsResponse(
```

Comment:
why was this needed?

Reply:
In the tests file we need to the object of the indices stats response. I have seen other response constructor were public only like ClusterStateResponse.